sdk/loadtestservice/azure-developer-loadtesting/swagger/README.md

```diff
@@ -1,37 +1,74 @@
 ## Generate autorest code
 
 ```yaml
-require: https://github.com/Azure/azure-rest-api-specs/blob/9401446f22177696d920cf110893a0de7452ee9e/specification/loadtestservice/data-plane/readme.md
+require: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/loadtestservice/data-plane/readme.md
```

Comment:
It's recommended to use a specific commit id instead of `main` branch.

Reply:
Updated to the commit ID of the GA swagger merge